<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="cursor" stdset="0">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="tabChangesFocus">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="file_new"/>
    <addaction name="file_open"/>
    <addaction name="file_save"/>
    <addaction name="separator"/>
    <addaction name="file_quit"/>
   </widget>
   <widget class="QMenu" name="edit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="edit_undo"/>
    <addaction name="separator"/>
    <addaction name="edit_copy"/>
    <addaction name="edit_cut"/>
    <addaction name="edit_paste"/>
   </widget>
   <widget class="QMenu" name="help">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
   </widget>
   <addaction name="file"/>
   <addaction name="edit"/>
   <addaction name="help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="file_new"/>
  </widget>
  <action name="file_open">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="file_new">
   <property name="icon">
    <iconset resource="manu.qrc">
     <normaloff>:/home/zz/work-src/github/qt/images/images/new.jpg</normaloff>:/home/zz/work-src/github/qt/images/images/new.jpg</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="file_save">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="file_quit">
   <property name="text">
    <string>退出(&amp;Q)</string>
   </property>
  </action>
  <action name="edit_copy">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="edit_paste">
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
  </action>
  <action name="edit_cut">
   <property name="text">
    <string>剪切(&amp;X)</string>
   </property>
  </action>
  <action name="edit_undo">
   <property name="text">
    <string>撤消(&amp;Z)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="manu.qrc"/>
 </resources>
 <connections/>
</ui>
